<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingWidget</class>
 <widget class="QWidget" name="TrainingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Training</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Train!</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Epoch :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="textBoxEpoch">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RMS error :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="textBoxError">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="plotButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
